librocketchat.c

```diff
@@ -23,6 +23,10 @@
 #define g_hash_table_contains(hash_table, key) g_hash_table_lookup_extended(hash_table, key, NULL, NULL)
 #endif /* 2.32.0 */
 
+#if !GLIB_CHECK_VERSION(2, 68, 0)
+#define g_memdump2 g_memdup

```

Comment:
Should this be `gmemdup2` instead of "dump"?

Reply:
Oh, also, needs a ` && !PURPLE_VERSION_CHECK(2, 14, 2)` because it accidentally(?) ended up in purple headers in 2.14.2

Reply:
Oh sorry typo.

What do you mean? I don't find the function in 2.14.2 sources.

Reply:
2.14.2 added a new libpurple/glibcompat.h which included a `static inline gpointer g_memdup2`